SPEC.md

```diff
@@ -492,7 +491,6 @@ The ACE must provide a Metadata service on the address given to the applications
 Clients querying any of these endpoints MUST specify the `Metadata-Flavor: AppContainer` header.
 
 [UUIDs](#pod-uuid) assigned to pods MUST be unique for a given instance of a metadata service.
-Hence, implementations of a metadata service shared among executors SHOULD either provide centralized allocation of UUIDs or partition the UUID space between individual executors.
```

Comment:
See previous comment about verbosity - I think we should provide examples because questions around the scope/scale of the metadata service have come up before.